pnpm-workspace.yaml

```diff
@@ -5,3 +5,4 @@ packages:
   - 'docs'
   - 'test'
   - 'apps/*'
+  - 'internal/*'
```

Comment:
This is where we will collect al internal packages?

Reply:
That's the idea. This could also be named `internal-packages` if you like, but `internal` matches the package prefix.

Reply:
👍  got it.

Up to you, I don't have a strong opinion on this. I we end up renaming it, I'd be more leaning towards `packages-internal` as that will place it next to the `packages` folder in the file explorer.